packages/jsii-rosetta/lib/languages/python.ts

```diff
@@ -608,19 +622,18 @@ export class PythonVisitor extends DefaultVisitor<PythonLanguageContext> {
     node: ts.StringLiteral | ts.NoSubstitutionTemplateLiteral,
     _context: PythonVisitorContext,
   ): OTree {
-    const rawText = node.text;
-    if (rawText.includes('\n')) {
+    if (node.getText(node.getSourceFile()).includes('\n')) {
```

Comment:
Whats this difference between this and node.text?

Reply:
`node.text` is the JavaScript value of the string (in particular, with escape sequences replaced with their literal value, e.g: `\n` will be a "real" newline), whereas `node.getText()` will return the corresponding source code (which will have the escape sequences untouched).

The change here improves the situation a bit (avoid multi-lining literals that were not actually multi-line in the source), but it's also not perfect (probably good enough?) as it'll still cause some escapes to be replaced within multi-line literals).